modules/api/src/main/java/net/countercraft/movecraft/CruiseDirection.java

```diff
@@ -0,0 +1,32 @@
+package net.countercraft.movecraft;
+
+import org.bukkit.block.BlockFace;
+
+public enum CruiseDirection {
+    NORTH, //0x3
+    SOUTH, //0x2
+    EAST, //0x4
+    WEST, //0x5
+    UP, //0x42
+    DOWN, //0x43
+    NONE;
```

Comment:
I'd like this to be backwards compatible wherein the enum just hides the raw byte values used previously behind fancy names for us humans.  (This would keep us from breaking any code we missed or any addons).

Reply:
Will do